content/php/concepts/string-functions/terms/str-contains/str-contains.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'str_contains()' 
+Description: 'Returns a boolean indicating if the specified string contains the substring provided.'
+Subjects: 
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags: 
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`str_contains()`** function performs a case sensitive search for a given substring within a string. Checking for the presence of an empty string will always return a positive result.
+
+## Syntax
+
+```pseudo
+str_contains($haystack, $needle)
+```
+
+- `$haystack` the string to be searched.
+- `$needle` the substring to be found.
+
+## Codebyte Example
```

Comment:
```suggestion
## Example
```

Reply:
I'm switching this to a standard code example because I think that the Codecademy compiler runs an older version of php that doesn't support this method.